scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -189,7 +191,7 @@ task ExtractTask {
     runtime {
         docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_d8a72b825eab2d979c8877448c0ca948fd9b34c7_change_to_hwe"
         memory: "12 GB"
-        disks: "local-disk 250 HDD"
+        disks: "local-disk 150 HDD"
```

Comment:
would be great to expose this as a parameter

Reply:
done